PortfolioProjectDay4Microverse

harshi0102 opened the pull request:
This project is made using HTML, CSS and Javascript

harshi0102 committed:
Update with the index.html and style.css from the previous branch

harshi0102 committed:
Update with the mobile hamburger corrections

harshi0102 committed:
Update with the recent work section and images

harshi0102 committed:
Merge branch 'main' into PortfolioProjectDay4Microverse

harshi0102 committed:
Update with the about section content in index.html

harshi0102 committed:
Merge branch 'PortfolioProjectDay4Microverse' of https://github.com/harshi0102/Portfolio-Microverse-project into PortfolioProjectDay4Microverse

harshi0102 committed:
Update with the about section

harshi0102 committed:
update with the index.html and style.css code to complete project

harshi0102 committed:
Update with the changes in the readme file and some corrections in the style.css

harshi0102 committed:
update with the changes again in style.css to fix corrections

harshi0102 committed:
Update with the readme file

harshi0102 committed:
Update with the corrections in the CSS file

harshi0102 committed:
update with the style.css corrections

harshi0102 committed:
Update with the corrections in styleint command

jasonfelice commented on index.html:
```diff
@@ -1,59 +1,425 @@
-    <header>
+  <!---Navigation-->
+  <section class="imagecontainer">
+ <header>
```
- [ ] Please make sure the hamburger remains in the same place as I resize the page.

## FIgma Design
![image](https://user-images.githubusercontent.com/92215878/220710658-a18e3bb4-4189-4a2b-9a93-5510902a1313.png)

## Your Design
![image](https://user-images.githubusercontent.com/92215878/220710767-b9f78917-637e-4edd-8e19-bfea5fbec5ba.png)

jasonfelice commented on index.html:
```diff
@@ -1,59 +1,425 @@
+  <div class="imgpatholder">
+<div class="imagebox">
+  <img src="image/Img Plaholder.png" alt="placeholder" >
+
```
- [ ] Please match the style of your cards to the given design.

## Figma Design
![image](https://user-images.githubusercontent.com/92215878/220711823-25818832-1ca8-4da1-a06d-ce1071985294.png)


## Your Design
![image](https://user-images.githubusercontent.com/92215878/220711674-e4c84475-111d-4934-8476-605b52cab3f3.png)

jasonfelice commented on index.html:
```diff
@@ -1,59 +1,425 @@
-        </nav>
-    </header>
-       
+<!--aboutme-->
```
- [ ] Please make sure here is perfectly aligned and following Figma style guidelines.

## Figma Design
![image](https://user-images.githubusercontent.com/92215878/220712681-bedf0d2d-2170-4f7e-a3b0-9578e1df3fa7.png)


## Your Design
![image](https://user-images.githubusercontent.com/92215878/220712519-9a446933-09ab-4c8a-8508-dfeafdabe5c8.png)

jasonfelice reviewed (changes requested):
Hi @harshi0102, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
No Linter errors✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

harshi0102 committed:
Update with the corrections in index.html and style.css

harshi0102 committed:
update with the corrections in the css file

harshi0102 commented:
Hi, I have made the below changes:
 hamburger remains in the same place as I resize the page, about me is aligned properly as per figma style, and cards design are also made as per figma style.

didierganthier commented on index.html:
```diff
@@ -1,59 +1,426 @@
+        <li><a href="#">CSS</a></li>
+        <li><a href="#">Javascript</a></li>
+        <li><a href="#">HTML</a></li>
+  </ul>
```
Well done on the project 👏🏽 and on making the required changes but your list of technologies wrapper lacks some padding for now. Consider adding some padding to the text so that it gives more space to the box. 📦 See below 👇🏽 

## Figma
![Screenshot 2023-02-23 at 06 48 06](https://user-images.githubusercontent.com/45739963/220897434-b89556d8-6672-4bf8-b106-a2628ebed3e5.png)

## Your design
![Screenshot 2023-02-23 at 06 48 23](https://user-images.githubusercontent.com/45739963/220897494-d024680a-8eac-42f9-b46c-3bcc0953cfb2.png)

This will greatly improve your design 😉

didierganthier commented on index.html:
```diff
@@ -1,59 +1,426 @@
+    </ul>
+  </div>
+  </div>
+</div>
```
This section looks great 🎉 so much hard work can be seen here, you're getting closer but not quite there yet, and your cards are not centered 📏 with each other. Take a look at your design 👇🏽 

![Screenshot 2023-02-23 at 06 50 43](https://user-images.githubusercontent.com/45739963/220898510-971ea10d-8a45-4ce3-897d-1021b4187ca3.png)

Also consider adding some spacing to the list of technologies here, you can use the `gap` attribute for that perhaps 🤔

didierganthier commented on index.html:
```diff
@@ -1,59 +1,426 @@
+      </div>
+    </div>
+
+      <br><br>
```
[OPTIONAL] I add this as optional as it's not part of the requirements but it would improve your project 💯 I suggest you avoid using the `<br>` tag because it's not good practice to do so ❌ , you might be wondering why? 🤔 The main reason for not using `<br>` is that it's not semantic. If you want two items in different visual blocks, you probably want them in different logical blocks. In most cases this means just using different elements, for example `<p>Stuff</p><p>Other stuff</p>`, and then using CSS to space the blocks out properly. All of that to say that I suggest using CSS to space the elements instead of the `<br>` tag. 👍🏽

didierganthier reviewed (changes requested):
Hi @harshi0102  ,

🎉 Great job on addressing the feedback from the previous review! You've made some solid progress in implementing the requested changes. However, there are still a few areas that need some attention before we can move on to the next project.

<img src="https://media.giphy.com/media/3oEduMcvxTNPlGuhH2/giphy.gif" />

## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

harshi0102 committed:
Update with the changes in style.css and index.html file

harshi0102 committed:
Update with the readme changes

harshi0102 commented:
I have added a padding according to the second feedback and added a gap also.I aligned the languages, frameworks skills container too as per the second feeback mentioned to me.Thanks

ridwanediallo commented on index.html:
```diff
@@ -279,7 +279,7 @@ <h1>About Me</h1>
 <div class="resumeinsideauto">
-<div class="resume" style="margin-left:122px;">
+<div class="resume" style="margin-left: 122px;">
 <button>Get My Resume</button>
```
- [ OPTIONAL ] I think adding a `    border-radius: 4px;` to your button would be better to make them look like the ones in the original design. See the comparison :point_down: 

### Your button 
![Screenshot from 2023-02-24 15-45-47](https://user-images.githubusercontent.com/89228743/221222745-19665030-6634-4978-952e-c04d2cba5d9b.png)

### The original design
![Screenshot from 2023-02-24 15-44-56](https://user-images.githubusercontent.com/89228743/221222722-3e5b9b15-a272-4cf8-aaca-3ef023a721f7.png)

- [ OPTIONAL ] And please for the last `Get in touch` button I would advise removing the border by giving it a `border: none`.

 ### Difference between your button and the original design :point_down: 
### Your button 
![Screenshot from 2023-02-24 15-49-43](https://user-images.githubusercontent.com/89228743/221224417-75b38cf0-8bd3-491a-b757-6c4910186594.png)

### The original design
![Screenshot from 2023-02-24 15-50-16](https://user-images.githubusercontent.com/89228743/221224445-fe5929d5-bb73-47ca-95a1-1f3e9e274022.png)

ridwanediallo reviewed (approved):
# STATUS: APPROVED 🟢
## Hi @harshi0102 , 
![Congratulations!GIF](https://user-images.githubusercontent.com/55918904/191301390-43d62dfa-bfdc-49c5-943b-4d72f4ea5bb8.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- Good commit messages. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
